src/app/shared/c-switch.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../injects';
+.c-switch{
```

Comment:
What does C-Switch mean? Please change it a clear and visible name, that explains. ALso cmn-toggle doesn't make sense, maybe you are trying to imply the common-toggle?